PurchasesTests/Attribution/AttributionFetcherTests.swift

```diff
@@ -0,0 +1,235 @@
+//
```

Comment:
this shows as a new file but it has been moved. There are new test cases at the bottom, I'll point out where they start.